.github/workflows/ci.yml

```diff
@@ -37,6 +37,23 @@ jobs:
           pip install black==19.10b0
           ./run-tests.sh --check-black
 
+  lint-flake8:
+    runs-on: ubuntu-latest
```

Comment:
Shall we switch to `ubuntu-18.04` or even better to `ubuntu-20.04`?  (See discussion on MM.)